include/freerdp/settings.h

```diff
@@ -1142,30 +1171,60 @@ struct rdp_settings
 	ALIGN64 UINT32 Password51Length;   /* 1281 */
 	ALIGN64 BOOL SmartcardLogon;       /* 1282 */
 	ALIGN64 BOOL PromptForCredentials; /* 1283 */
-	UINT64 padding1344[1344 - 1284];   /* 1284 */
+
```

Comment:
The settings should be reworked, currently there are just too many of them just for kerberos.
Maybe just pass on the command line to the module?